starknet_devnet/devnet_config.py

```diff
@@ -264,6 +264,16 @@ def parse_args(raw_args: List[str]):
         action="version",
         version=__version__,
     )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        help="Show more verbose logs",
+    )
+    parser.add_argument(
+        "--hide-logs",
+        action="store_true",
+        help="Hide info logging",
```

Comment:
I think it is not clear what we mean by "info logging". Do you mean the server access logging?

Reply:
Yes. I decided to add flag and not hide access logs without intention, cause they are useful

Reply:
Ok, can you just change the `help` string then, perhaps by replacing "info logging" with "server access logging"?